app-portage/gentle/gentle-1.0.0.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 2023 Gentoo Authors
+# Copyright 2023-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
-PYTHON_COMPAT=( python3_{10..11} )
+PYTHON_COMPAT=( python3_{11..12} )
```

Comment:
Why drop python 3.10?

Reply:
The code imports `tomllib` without `try/except` now. Python 3.10 [will be EOL](https://projects.gentoo.org/python/guide/interpreter.html) in Gentoo around June/July.